coop_membership/views/view_res_partner.xml

```diff
@@ -585,7 +600,7 @@
             </button>
             <xpath expr="//notebook/page[1]/field/form/sheet/group//field[@name='title']"
                    position="after">
-                <field name="sex" attrs="{'required': True}"/>
+                <field name="gender" attrs="{'required': True}"/>
             </xpath>
```

Comment:
Again.. this field is already in https://github.com/OCA/partner-contact/blob/12.0/partner_contact_gender/views/res_partner.xml.